### ISSUE TITLE
Multi-command pipes

### ISSUE THREAD

**Issue by MoritzWM** [tomotools/tomotools#12]
The batch-prepare-tiltseries and reconstruct functions are getting more and more complicated and have more and more options. Maybe I found an alternative approach that is easier to handle:
We could build [multi-command pipelines](https://click.palletsprojects.com/en/8.1.x/commands/#multi-command-pipelines). An example is shown [here](https://github.com/pallets/click/tree/main/examples/imagepipe).

In our case, this could look something like:
`tomotools motioncor2 <path_to_frames> align-with-aretomo --local ctfplotter dose-filter reconstruct-with-imod --trim <output_dir>`

In essence, each function would do one thing (motioncor2, align-with-aretomo, align-with-imod, dose-filter) and accept options for itself. They could be chained, each taking as input the output of the previous command.
I don't know how difficult this would be to implement. I think for this we really have to dig into the click documentation, but it could be worth it.

**Comment by bwmr** [comment]
Sounds like a good option for custom pipelines! I think we should maintain the basic commands as they are now for "basic" usage though...

**Comment by bwmr** [comment]
@MoritzWM can this be closed as inactive?

**Comment by MoritzWM** [comment]
Yeah it can. I have this very feature in tomojob, there it's super convenient. But here I think it's not needed...
